HW04.Rmd

```diff
@@ -78,42 +151,114 @@ Assume that:
 ### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
 
 ```{r race}
-
+#select data and count
+race_intent <- gun_deaths %>% 
+    select(intent, race) %>% 
+    filter(intent %in% c("Suicide", "Homicide"), race %in% c("White", "Black", "Hispanic")) %>% 
+    group_by(race, intent) %>% 
+    count() %>% 
+    ungroup() %>% 
+    group_by(race) %>% 
+    mutate(total_race = sum(n)) %>% 
+    mutate(percentage = round(n / total_race * 100, digits = 2)) %>% 
+    select(-total_race)

```

Comment:
This is a neat way for calculating the percentage and adding a separate "percentage" column to your data frame!